File: files.operations.ui/src/main/java/l/files/operations/ui/OperationsUi.java
package l.files.operations.ui;

import android.content.Context;
import android.support.v4.content.LocalBroadcastManager;

import l.files.operations.ui.notification.NotificationReceiver;

public final class OperationsUi {
  private OperationsUi() {}

  /**
   * Initializes this module on application startup.
   */
  public static void init(Context context) {
    NotificationReceiver.register(LocalBroadcastManager.getInstance(context));
  }
}

File: operations-ui/src/main/kotlin/l/files/operations/ui/OperationsUi.kt
package l.files.operations.ui

import android.content.Context
import l.files.operations.ui.notification.NotificationReceiver
import l.files.operations.Events

public class OperationsUi {

  public fun init(context: Context) {
    Events.get()!!.register(NotificationReceiver(context))
  }

}
